setup.py

```diff
@@ -42,7 +69,7 @@ def main():
     if version is None:
         import versioneer
         version = versioneer.get_version()
-        cmdclass = versioneer.get_cmdclass()
```

Comment:
@effigies not sure this is really necessary - WDYT?

Reply:
This updates the `_version.py` to just be a dictionary that gets returned, rather than a massive Python file to generate that dictionary. I would recommend keeping that.

Reply:
I'm bringing this back, and removing the `VERSION` hack